packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/TxDetailModal.tsx

```diff
@@ -92,8 +88,10 @@ export const TxDetailModal = ({ tx, rbfForm, onCancel }: TxDetailModalProps) =>
     const confirmations = useSelector(state =>
         selectTransactionConfirmations(state, tx.txid, accountKey),
     );
-    const account = useSelector(state => selectAccountByKey(state, accountKey));
-    const network = account && getAccountNetwork(account);
+    const account = useSelector(state => selectAccountByKey(state, accountKey)) as Account;
+    const network: Network = networks[account.symbol];
```

Comment:
This is becoming a widespread pattern - that I access `networks` using a generic `NetworkSymbol`, and need to widen the type as `Network` (it is a generic, unspecified network), instead of inferred type, which is union of all specific networks.

The inferrence is really great when accessing with specific `NetworkSymbol` such as `networks.btc`, we really don't want to lose that..

:point_right: should I create a helper function that widens the type? like
`getNetwork = (symbol: NetworkSymbol): Network => networks[symbol]`

Reply:
Yeah, `'rbf'` is not assignable to `never`, because _some of the networks_ don't have it :see_no_evil: 
I'll do the util in another PR :slightly_smiling_face: